The busybox executable can show itself in the --list output, which will
overwrite the executable previously copied with a self-referential symlink. By
copying the executable *after* copying all the symlinks, we ensure that the
problem is overwritten.

--- a/install/base
+++ b/install/base
@@ -4,12 +4,12 @@
 build() {
     local applet
 
-    add_binary /usr/lib/initcpio/busybox /bin/busybox
-
     for applet in $(/usr/lib/initcpio/busybox --list); do
         add_symlink "/usr/bin/$applet" busybox
     done
 
+    add_binary /usr/lib/initcpio/busybox /bin/busybox
+
     # add kmod with applet symlinks
     add_binary kmod
     for applet in {dep,ins,rm,ls}mod mod{probe,info}; do
